lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -21,13 +22,21 @@ class Submit
       include Sidekiq::Job
       sidekiq_options retry: 3
 
+      STATSD_KEY_PREFIX = 'worker.evss.submit_form526_backup'
+
+      # :nocov:
+      sidekiq_retries_exhausted do |msg, _ex|
+        job_exhausted(msg, STATSD_KEY_PREFIX)
+      end
+
       def perform(form526_submission_id)
         return unless Settings.form526_backup.enabled
 
-        job_status = Form526JobStatus.create!(job_class: 'BackupSubmission', status: 'pending',
-                                              form526_submission_id:, job_id: jid)
-        Processor.new(form526_submission_id).process!
-        job_status.update(status: Form526JobStatus::STATUS[:success])
+        submission = Form526Submission.find(form526_submission_id)
+        with_tracking('Form526 Backup Submission', submission.saved_claim_id, submission.id) do
+          Processor.new(form526_submission_id).process!
+          job_status.update(status: Form526JobStatus::STATUS[:success])
+        end
       rescue => e
         ::Rails.logger.error(
           message: "FORM526 BACKUP SUMBISSION FAILURE. Investigate immedietly: #{e.message}.",
```

Comment:
I admire your restraint in NOT fixing these typos 🤣

Reply:
oh i just didn't notice lol